src/main/java/com/google/sps/Form.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps;
+
```

Comment:
I think this file is fulfilling the same purpose as the Event.java and Location.java files that Joshua had made already.